### ISSUE TITLE
Sorting Bug - When Adding a New Record

### ISSUE THREAD

**Issue by barahonam1** [handsontable/handsontable#425]
To reproduce this bug, please follow these steps: 

1. Go to http://jsfiddle.net/DHEvR/
2. Sort column A.
3. Add new values below that column.

Up until now you can see that the scroll bar does not detect the new values automatically. Then if you add new values under column B, the first value is displayed, the second and subsequent values do not display (not until you scroll up and down every time). Then if you randomly scroll all the way up and then all the way down the values get messy and some recently added values even disappear. In short the table exhibits a strange behavior.

**Comment by AMBudnik** [comment]
This demo doesn't render handsontable. If you have any questions please open new issue.
